docs/source/user_guide/release_notes.md

```diff
@@ -1,4 +1,38 @@
 # Release Notes
+## v0.13.0rc1 - 2025.12.27
+This is the first release candidate of v0.13.0 for vLLM Ascend. We landed lots of bug fix, performance improvement and feature support in this release. Any feedback is welcome to help us to improve vLLM Ascend. Please follow the [official doc](https://vllm-ascend.readthedocs.io/en/latest) to get started.
+
+### Highlights
+- Improved the performance of DeepSeek V3.2. [#3995](https://github.com/vllm-project/vllm-ascend/pull/3995)
```

Comment:
, please refer to [tutorials](https://docs.vllm.ai/projects/ascend/en/latest/tutorials/DeepSeek-V3.2.html)